Userland/Applications/TextEditor/MainWidget.cpp

```diff
@@ -475,6 +475,18 @@ ErrorOr<void> MainWidget::initialize_menubar(GUI::Window& window)
 
     view_menu->add_separator();
 
+    view_menu->add_action(GUI::CommonActions::make_zoom_in_action([&](auto&) {
+        auto& font = m_editor->font();
+        auto new_size = max(5, min(40, font.presentation_size() + 1));
```

Comment:
I believe we have a `clamp` method that is a bit more digestible than `max(x, min(y, z))`.